VoiceOfAmerica/ViewControllers/VoteViewController.swift

```diff
@@ -60,5 +63,10 @@ class VoteViewController: UIViewController, UITableViewDelegate, UITableViewData
         tableView.edgeAnchors == safeAreaGuide.edgeAnchors
 
     }
+
+    func refreshData() {
+        self.candidates = model?.getCandidates() ?? [Candidate]()
```

Comment:
#req Let's use this, so that we don't clear out users' current information if they lose network connection.

```suggestion
        if let newCandidates = model?.getCandidates() {
                self.candidates = newCandidates
                tableView.reloadData()
        }
```

Reply:
Cool